CODEOWNERS

```diff
@@ -87,3 +87,4 @@ NOTICES     @garyverhaegen-da @dasormeter
 /libs-scala/nonempty-cats/src/    @S11001001
 /libs-scala/scala-utils/src/    @S11001001
 /libs-scala/scalatest-utils/src/    @S11001001
+/libs-scala/test-evidence/ @digital-asset/kv-participant @soren-da @matthiasS-da
```

Comment:
Looks like there's an error here. Does @soren-da exist? Does he have write access?

Reply:
Looks like @soren-da exists but doesn't have write access to the repo.

Reply:
I have already asked him, he will request an access and then this PR can be merged.

Reply:
he should be added now